i18n/factory/translator.js

```diff
@@ -203,6 +203,13 @@ module.exports = ['$rootScope', '$http', function ($rootScope, $http) {
 
         },
 
+        setDataForLanguage: function setDataForLanguage(data, language) {
+
+            if(config.lazy) {
+                i18n.data[language] = data;
```

Comment:
Because i have to add data for a language after init and it doesn't work with the current code. I can't set the language dynamically